# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1245080386 14400
#      Mon Jun 15 11:39:46 2009 -0400
# Node ID 52bde0edd23a46ccae022c7b26b04fe6735aece2
# Parent  335dc62068a80dc72d1056c871206a02025af756
quiver.m: Add 'clf' to demos.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,10 +1,11 @@
 2009-06-15  Ben Abbott <bpabbott@mac.com>
 
+	* plot/quiver.m: Add 'clf' to demos.
 	* plot/plotyy.m: Consider dataaspectratiomode before changing
 	dataaspectratio.
 
 2009-06-14  Ben Abbott <bpabbott@mac.com>
 
 	* plot/plotyy.m: Fix compatibility with subplot, add listeners for
 	dataaspectratio, and add a demo.
 
diff --git a/scripts/plot/quiver.m b/scripts/plot/quiver.m
--- a/scripts/plot/quiver.m
+++ b/scripts/plot/quiver.m
@@ -77,16 +77,17 @@ function retval = quiver (varargin)
 
   if (nargout > 0)
     retval = tmp;
   endif
 
 endfunction
 
 %!demo
+%! clf
 %! [x,y] = meshgrid(1:2:20);
 %! h = quiver(x,y,sin(2*pi*x/10),sin(2*pi*y/10));
 %! set (h, "maxheadsize", 0.33);
 
 %!demo
 %! axis("equal");
 %! x=linspace(0,3,80); y=sin(2*pi*x); theta=2*pi*x+pi/2;
 %! quiver(x,y,sin(theta)/10,cos(theta)/10);
